Resources/Private/Public/JavaScript/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
Resources/Private/Public?!?!

Reply:
I removed this. I added this folder with an incorrect build taken one folder earlier